### PR TITLE
Cli tests type annotations

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- add type annotations for cli tests
- remove unused imports

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: tests/cli/conftest.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
+from typing import Callable, Any, Tuple
 import pytest
 import os
+from _pytest.pytester import Testdir, RunResult
 
 
 @pytest.fixture
-def run(testdir):
+def run(testdir: Testdir) -> Callable[[Tuple[Any]], RunResult]:
```
Comment (federicotdn) [inline tests/cli/conftest.py]: Is there any particular reason we're not using `Callable[..., RunResult]` instead of `Callable[[Tuple[Any]], RunResult]`?

Comment (wochinge) [reply tests/cli/conftest.py]: Nope, gonna fix it

File: tests/cli/test_rasa_init.py [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import os
+from typing import Callable, Any, Tuple
+from _pytest.pytester import RunResult
 
 
-def test_init(run):
+def test_init(run: Callable[[Tuple[Any]], RunResult]) -> None:
```
Comment (federicotdn) [inline tests/cli/test_rasa_init.py]: I noticed that some tests have a return type and others don't. I think we should decide for one option and stick to that one. On a PR for Rasa X Rick mentioned that he's not including them: https://github.com/RasaHQ/rasa-x/pull/1376#discussion_r334285027.

Comment (wochinge) [reply tests/cli/test_rasa_init.py]: ok, will also remove them then 👍

**Pull request merged**